### ISSUE TITLE
NVDA-alpha-28685 caused some add-ons to become incompatible

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#15141]
### Steps to reproduce:
1. Install NVDA-Alpha-28685
2. Install "Instant Translate" or "UnigramPlus" add-on.
3. Restart NVDA
### Actual behavior:
The add-on failed to load because the settings panel failed to initialize.
Try typing `from gui import SettingsPanel` in the NVDA Python console and execute it.
```log
Traceback (most recent call last):
  File "<console>", line 1, in <module>
ImportError: cannot import name 'SettingsPanel' from 'gui' (C:\Program Files (x86)\NVDA\library.zip\gui\__init__.pyc)
```
### Expected behavior:
Add-on compatibility should not be broken
### NVDA logs, crash dumps and other attachments:
[log.txt](https://github.com/nvaccess/nvda/files/12049659/log.txt)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA-Alpha-28685
#### Windows version:
Windows 10 22H2 (AMD64) build 19045.3155
#### Name and version of other software in use when reproducing the issue:
None
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
alpha-28675 can't reproduce this, I suspect it's a regression of #15121.
#### If NVDA add-ons are disabled, is your problem still occurring?
No
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by XLTechie** [comment]
I can confirm this.

Related: #15142

**Comment by beqabeqa473** [comment]
I will update my addons to be compatible
